src/components/molecules/EarthEditor/DatasetPane/DatasetModal/hooks.ts

```diff
@@ -2,28 +2,44 @@ import { useCallback, useState, useEffect } from "react";
 import { useIntl } from "react-intl";
 import useFileInput from "use-file-input";
 import { Type as NotificationType } from "@reearth/components/atoms/NotificationBar";
+import { SheetParameter } from "./Gdrive/hooks";
```

Comment:
Re-export SheetParameter at `./Gdrive/index.tsx` for SheetParameter to enable to be imported from `./Gdrive`.